### PR TITLE
WINC clarify Getting Support wording

### PR DESCRIPTION
**Pull request by mburke5678**
Per [Slack conversation](https://coreos.slack.com/archives/CM4ERHBJS/p1648832040363479?thread_ts=1648789414.418059&cid=CM4ERHBJS)"
"https://docs.openshift.com/container-platform/4.9/windows_containers/windows-containers-release-notes-4-x.html#getting-support is confusing,...  I thus recommend fixing docs and pointing to https://access.redhat.com/support/policy/updates/openshift#windows" 
 
The wording in the PR is  wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows

 Previews: 
[Getting support](https://deploy-preview-44201--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-5-x.html#getting-support)
[OKD without the Getting support section](https://deploy-preview-44201--osdocs.netlify.app/openshift-origin/latest/windows_containers/windows-containers-release-notes-5-x).
 
 Changing 4.6 to 4.9 in separate PRs, as this section is in the release notes and only in that branch.

### CODE REVIEW

**Comment by mburke5678** [comment]
@rrasouli Apologies. 

Here is a preview of [OKD with the _Getting support_ section](https://deploy-preview-44201--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-4-x.html#getting-support)
Here is a preview of [OKD without the _Getting support_ section](https://deploy-preview-44201--osdocs.netlify.app/openshift-origin/latest/windows_containers/windows-containers-release-notes-4-x).

**Comment by mburke5678** [comment]
These changes started in https://github.com/openshift/openshift-docs/pull/44112.

**Comment by kelbrown20** [comment]
LGTM!

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 1]
```diff
@@ -13,10 +13,20 @@ Windows Container Support for Red Hat OpenShift enables running Windows compute
...
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for the Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
```
Comment (stevsmit) [inline windows_containers/windows-containers-release-notes-5-x.adoc:24]: is "the" necessary before the name of the product in this sentence: 

You must have this separate subscription to receive support for **the** Windows Container Support for Red Hat OpenShift.

**Pull request merged**